yt_dlp/extractor/youtube.py

```diff
@@ -2445,19 +2440,23 @@ def _real_comment_extract(contents):
         }
 
     @staticmethod
-    def _generate_player_context(sts=None):
+    def _get_checkok_params():
+        return {'contentCheckOk': True, 'racyCheckOk': True}
+
+    @classmethod
+    def _generate_player_context(cls, sts=None):
         context = {
             'html5Preference': 'HTML5_PREF_WANTS',
         }
         if sts is not None:
             context['signatureTimestamp'] = sts
-        return {
+        playback_context = {
             'playbackContext': {
                 'contentPlaybackContext': context
-            },
-            'contentCheckOk': True,
-            'racyCheckOk': True
+            }
         }
+        playback_context.update(cls._get_checkok_params())
+        return playback_context
```

Comment:
```suggestion
        return {
            'playbackContext': {
                'contentPlaybackContext': context
            },
            **cls._get_checkok_params()
        }
```
nothing wrong with your code, just a suggestion